### PR TITLE
Update demo.rst

### PR DESCRIPTION
**Pull request by ksens**
* Fixed error in Python code snippet

### CODE REVIEW

File: docs/demo.rst [hunk 1]
```diff
@@ -273,7 +273,7 @@ around 3GB. Next, we will add the reference set.
 .. code-block:: bash
 
     $ ga4gh_repo add-referenceset registry.db /full/path/to/hs37d5.fa.gz \
-      -d “NCBI37 assembly of the human genome” --ncbiTaxonId 9606 --name NCBI37 \
+      -d "NCBI37 assembly of the human genome" --ncbiTaxonId 9606 --name NCBI37 \
```
Comment (david4096) [inline docs/demo.rst]: Good catch!

File: docs/demo.rst [hunk 2]
```diff
@@ -185,7 +185,7 @@ performed above, we can use the following code:
 
     from __future__ import print_function
 
-    import ga4gh.client.client as client
+    import ga4gh.client as client
```
Comment (david4096) [inline docs/demo.rst]: The signature has changed in the latest version. I personally prefer `from ga4gh.client import client`, but the line you've removed is correct for the current development version. 0.3.4 released version uses the syntax you've provided.

**Comment by kozbo** [comment]
-1 Actually that is the correct way to import the client in the new ga4gh-client namespace. Think that these docs are a little ahead of the released code. We are in the middle of a release now which should be consistent with this doc. (sorry for posting this in the wrong PR earlier. Having both PR's have the same name is confusing)

**Comment by ksens** [comment]
@david4096 do take a look now.

**Comment by david4096** [comment]
Thanks @ksens !

**Pull request merged**